jetcd-core/src/main/java/com/coreos/jetcd/internal/impl/ClientConnectionManager.java

```diff
@@ -159,7 +160,9 @@ synchronized void close() {
         .nameResolverFactory(
```

Comment:
it seems to be setting `nameResolverFactory` again is not needed. 
`defaultChannelBuilder()` has already set the `nameResolverFactory`.

Reply:
There is a small difference between the two: 
- `defaultChannelBuilder` uses the endpoints from the builder
- `withNewChannel` provides its own endpoint so we need to override the `nameResolverFactory`